perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -206,6 +206,32 @@ sub user_from_oidc {
 
 =cut
 
+=head2 dashboard_export_problems_add_columns
+
+Brent have an additional column for street name, fetched using OSPlaces API.
+
+=cut
+
+sub dashboard_export_problems_add_columns {
+    my ($self, $csv) = @_;
+
+    return unless $self->get_reverse_geocoder eq 'OSPlaces';
+
+    $csv->add_csv_columns(
+        street_name => 'Street Name',
+    );
+
+    $csv->csv_extra_data(sub {
+        my $report = shift;
+
+        my $closest_address = $self->find_closest($report);
```

Comment:
We shouldn't call this in a CSV export because it may involve external lookups. See how Hackney calls the report's `nearest_address` function instead to get the data if it's already present. `parts->street` would be what you want here.